addons/website_sale_slides/static/src/xml/website_slides_unsubscribe.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-extend="website.slides.unsubscribe.modal.leave">
+        <t t-jquery="p" t-operation="after">
+            <t t-if="widget.enroll === 'payment'">
+                <p class="alert alert-warning">
+                    <i class="fa fa-exclamation-triangle fa-3x float-left mr-3"></i>
+                    This course needs payment.<br/>
+                    If you leave the course and then want to join again, you will have to pay again!
```

Comment:
`If you leave the course by mistake, you are fucked !` would also be a great warning :smiley:

Reply:
@dbeguin a bit offensive, don't you think.
I suggest: 
`This course is paid. <br>`
`Leave the course and re-enroll afterwards will be charged.`

Reply:
@sbuhl not a good sentence. Then it should be:
"Leaving the course and re-enrolling afterwards means that you'll be charged again" or something

Reply:
We have a winner! Thanks @Yenthe666 :-)

Reply:
🎉 awesome, thanks!

Reply:
> @dbeguin a bit offensive, don't you think.
> I suggest:
> `This course is paid. <br>`
> `Leave the course and re-enroll afterwards will be charged.`

This 'you are fucked part' was a joke, of course, my dear @sbuhl :)